data-service/service/oracle_databases.go

```diff
@@ -45,11 +46,32 @@ func (hds *HostDataService) oracleDatabasesChecks(previousHostdata, hostdata *mo
 
 	hds.checkNewLicenses(previousHostdata, hostdata, licenseTypes)
 
+	var unlistedDatabasesAlerts []model.Alert
+
 	for _, dbname := range hostdata.Features.Oracle.Database.UnlistedRunningDatabases {
 		if err := hds.ackOldUnlistedRunningDatabasesAlerts(hostdata.Hostname, dbname); err != nil {
 			hds.Log.Errorf("Can't ack UnlistedRunningDatabases alerts by filter")
 		}
-		if err := hds.throwUnlistedRunningDatabasesAlert(dbname, hostdata.Hostname); err != nil {
+
+		unlistedDatabasesAlerts = append(unlistedDatabasesAlerts,
+			model.Alert{
+				ID:                      primitive.NewObjectIDFromTimestamp(hds.TimeNow()),
```

Comment:
Since this parte from line 58 to 64 is replicated, put it in the method `throwUnlistedRunningDatabasesAlert` instead!